src/imageResponse.js

```diff
@@ -56,16 +56,16 @@ const redirectImageToWithinBounds = (params, response) => {
 };
 
 const redirectToCachedEntity = (cacheUrl, params, response) => {
+  // We redirect the user to the new location. The 307 ensures future requests come in to this service anyway
+  // 307 redirects are not cached by browsers
   response.status(307).set({
-    'Location': cacheUrl,
-    'Cache-Control': 'public',
```

Comment:
Yup. Plus we do not want it to be cached forever, since that would now allow us to ever clear our caches.

The users with cached urls would be directed to a non-existent object, and we would have to hope a user comes by without the cache so the image will be re-created.